lib/cocoapods/target/aggregate_target.rb

```diff
@@ -292,7 +292,7 @@ def resource_paths_by_config
                 extname = File.extname(resource_path)
                 if self.class.resource_extension_compilable?(extname)
                   output_extname = self.class.output_extension_for_resource(extname)
-                  built_product_dir.join(File.basename(resource_path, extname)).sub_ext(output_extname).to_s
```

Comment:
@dnkoutso 
This is actually unnecessary. I realize I've used `sub_ext` incorrectly in my impl.  
The correct use would be: 
```
built_product_dir.join(resource_path).sub_ext(output_extname).to_s
```